docs/design/node-diagram.md

```diff
@@ -0,0 +1,13 @@
+# Node Diagram
+
+This page depicts the node diagram designs for Autoware Core/Universe architecture.
+
+## Autoware Core
+
+TBD.
+
+## Autoware Universe
+
+![Node diagram](node-diagram/overall-node-diagram-autoware-universe.drawio.svg)
+
+[Open in draw.io for fullscreen](https://app.diagrams.net/#Htier4%2Fautoware-documentation%2Fadd-node-topic-architecture%2Fdocs%2Fdesign%2Fnode-diagram%2Foverall-node-diagram-autoware-universe.drawio.svg)
```

Comment:
I guess you need to fix here.

```suggestion
[Open in draw.io for fullscreen](https://app.diagrams.net/#Htier4%2Fautoware-documentation%2Ftier4/main%2Fdocs%2Fdesign%2Fnode-diagram%2Foverall-node-diagram-autoware-universe.drawio.svg)
```

Reply:
(Perhaps `/` cannot be used.)

Reply:
done. https://github.com/tier4/autoware-documentation/pull/10/commits/c3ab3a8d35c1ff8b848673d7e4c3ed4e6c0b6d2f